app/routes/appendix/appendix.js

```diff
@@ -21,7 +21,7 @@ const updateSchema = schemaGenerator(db.models.templateAppendix, {
 router.use('/', async (req, res, next) => {
   try {
     if (!req.body.projectId && !req.body.templateId) {
-      req.templateAppendix = await db.models.templateAppendix.scope('public').findAll();
+      req.templateAppendix = await db.models.templateAppendix.scope('extended').findAll();
```

Comment:
you don't need to add a scope at this point, only before returning the body to the user, as we may need to do some pre-processing before returning it. See line 43 and 96 at app/routes/template/templateAppendix.js